code/game/machinery/doors/firedoor.dm

```diff
@@ -285,6 +285,8 @@
 
 	if(environment.temperature >= FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
 		return FIRELOCK_ALARM_TYPE_HOT
+	if(environment.gases[/datum/gas/antinoblium] && environment.gases[/datum/gas/antinoblium][MOLES] > MINIMUM_MOLE_COUNT)
+		return FIRELOCK_ALARM_TYPE_HOT
```

Comment:
this is kinda hacky ngl

Reply:
It is. I think I detected some strange issue where the firelock would rapidly open and close when I tried to make it do its own thing. I'll investigate that later.

Reply:
it's likely heat spiking and then spreading out. I can't recall if we have a grace window between high and low but maybe we should